applications/GeoMechanicsApplication/tests/test_curved_beam_elements.py

```diff
@@ -0,0 +1,42 @@
+import sys
+import os
+
+sys.path.append(os.path.join('..', '..', '..'))
+sys.path.append(os.path.join('..', 'python_scripts'))
```

Comment:
I STRONGLY recommend against doing this
Modifying the system path remains after running the test and might lead to some hard-to-detect import problems!

Why do you need this?
You should be able to import all the python scripts in the usual way